# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/FailedCertSecurityInfo.webidl
# Commit: 665058a3a942
# Full Hash: 665058a3a9421067f266e839a5a6e56ddfaf41d4
# Author: prathiksha <prathikshaprasadsuman@gmail.com>
# Date: 2019-06-27 09:34:48
# Regressor Bug: 1555438
# File Overlap Count: 1
# Description:
#   Bug 1555438 - Expose attributes from nsISiteSecurityService and failedCertChain info to content for about:certerror use. r=baku
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D33064
# ==============================================================================

diff -r 967f87a35045 -r 665058a3a942 dom/webidl/FailedCertSecurityInfo.webidl
--- a/dom/webidl/FailedCertSecurityInfo.webidl	Wed Jun 26 20:23:42 2019 +0000
+++ b/dom/webidl/FailedCertSecurityInfo.webidl	Wed Jun 26 20:23:55 2019 +0000
@@ -20,4 +20,8 @@
   DOMString issuerCommonName = "";
   DOMTimeStamp certValidityRangeNotAfter = 0;
   DOMTimeStamp certValidityRangeNotBefore = 0;
+  DOMString errorMessage = "";
+  boolean hasHSTS = true;
+  boolean hasHPKP = true;
+  sequence<DOMString> certChainStrings;
 };
